docs/initiatives/personal-website-club.mdx

```diff
@@ -9,26 +9,61 @@
   @Personal Website Builders Club
 </div>
 
-Members of the Personal Website Club meet periodically to work on their personal websites.
+## Overview
 
-While there is no specific agenda for the club's reunions, members gather together to
-collectively work, cheer and help each other, or to get accountability for the progress they
-wish to make.
+Members of the *Personal Website Club* meet periodically to work on their personal websites.
 
-Since Fandom Coders experienced in wed development are often present during Personal Website Club
-days, they can be the perfect occasions for beginners to get some focused practice and support.
+While there is no specific agenda for the club's meetings, members gather together to collectively work, cheer each other on, offer help, or get accountability for their goals and progress.
 
-All Fandom Coders members are welcome to participate in "Personal Website Club" days,
-regardless of role assignment.
+Since members of Fandom Coders who are experienced in web development often participate in Personal Website Club days, these events can be the perfect occasions for beginners to get some focused practice, support, and advice from fellow club members.
 
-## Calling for a "Personal Website Club" day
+All Fandom Coders members are welcome to participate in Personal Website Club days regardless of role assignment.
 
-There is currently no fixed schedule to the "Personal Website Club", and every member is free
-to call for a new day to be organized (or organize it themselves).
+## Schedule
+
+Officially, the Personal Website Club meets **every week on Saturdays**, and each event runs all day to accomodate members from different time zones.
+
+## Scheduling a *Personal Website Club* day
 
 There are two types of Personal Website Club events:
 
-- **Scheduled:** These days are organized in advance and added to the Discord event calendar. A survey
-  might be run to find a day and time that works for most members.
-- **Impromptu:** Members who want to work on their own personal website can simply declare such a day and
-  see if others want them to join on their endeavor.
+- **Scheduled:** These days are organized in advance and added to the Discord event calendar. 
+- **Impromptu:** Members who want to work on their own personal website can simply declare such a day and see if others want them to join on their endeavor. You can declare an impromptu Website Club meeting in the *#personal_website_club* channel on the Discord server by simply announcing it.
+
+## Featured projects from members of the *Personal Website Club*
+
+If you are a member of the Website Club and would like your site to be featured here, check out the pinned messages in the *#personal_website_club* channel to find out how to submit yours!
+
+***
```

Comment:
Can you file an issue to move these to their own page, or at least not have the information just written here, but have a file with all the sites info that is then pulled into the page and display? I can explain better in private if this isn't clear.

Anyway, once the issue is filed, I'd add something here like:

```
:::todo
These projects should be moved to their own page! Want to help? Check out this issue (link).
:::
```

Reply:
As discussed, we're going to work on this together now.